### DISCUSSION TITLE
Perhaps an Icaros Lite for HEVC files

### DISCUSSION THREAD

**Discussion by HHH0003** [Xanashi/Icaros#72] [Feature Requests]
The new Windows 11 23H2 File Explorer has difficulty with HEVC thumbnails.  After my computer updated to 23H2, I noticed some of my iPhone HEVC movies had a "green mist" and "pixelization" on the thumbnails.  I thought it was the HEVC codec and tried Shark 007 and K-Lite codecs to fix the problem.  These programs did fix the thumbnail issue... it was because both use Icaros for thumbnail generation.  Whatever the issue is with the new File Explorer, Icaros does fix the problem with the HEVC thumbnails.  Perhaps a small program for HEVC files only?  Just a thought.

**Comment by Xanashi** [comment]
Icaros is already a light software, so I would simply recommend to people who just want HEVC fixed, to install the [Icaros directly](https://github.com/Xanashi/Icaros/releases/tag/v3.3.1), then clear the thumbnailing filetypes box on the 'Thumbnailing' page and only add the formats that they want to be fixed. Finally enable Thumbnailing.

It is also possible to simply drag and drop files unto the 'Thumbnailing' page in Icaros to add their filetype.

**Reply by HHH0003** [reply]
I discovered using Icaros properties handler disables any GPS information included in the image.  Could this be modified and allow GPS information to be included?  

I included two screenshots of the HEVC images Windows 23H2 is having issue with. Icaros does fix this problem.  I've noticed OneDrive is Pixelating some HEVC images... but doesn't create the green mist.   
![Screenshot 2023-10-09 131014](https://github.com/Xanashi/Icaros/assets/95982941/ce3b7fcb-aeb7-436a-aef3-f2198de791bc)
![Screenshot 2023-10-09 131115](https://github.com/Xanashi/Icaros/assets/95982941/410d8396-f399-4f46-93df-9bb5c50bc12e)

**Reply by Xanashi** [reply]
Could you send me a sample file with GPS info. Could you also show a screenshot of the GPS info when it's shown.

OneDrive (I believe) uses its own thumbnail provider, and is not affected by the Windows thumbnail providers, which is most likely
why you're seeing different results for files stored on it.

**Reply by HHH0003** [reply]
### Without Icaros properties enabled...
### No enhanced information.
![Screenshot 2023-10-11 123017](https://github.com/Xanashi/Icaros/assets/95982941/b474ea25-00fe-4672-becf-c940ed90460e)
### GPS Location is shown.
![Screenshot 2023-10-11 123112](https://github.com/Xanashi/Icaros/assets/95982941/00ca981b-cb48-417e-8824-94a7efd47747)

### With Icaros properties enabled.
### Enhanced information shown.

![Screenshot 2023-10-11 123144](https://github.com/Xanashi/Icaros/assets/95982941/90ecca77-df8a-4803-affe-830d3e1acf1c)
### GPS location is missing...
![Screenshot 2023-10-11 123613](https://github.com/Xanashi/Icaros/assets/95982941/8a3907bd-63b5-4e6e-9bf3-db43aadcff77)

**Reply by Xanashi** [reply]
Thank you. 
Would it also be possible for you to provide a sample file?

**Reply by HHH0003** [reply]
### Hopefully this works...

[](url)https://drive.google.com/file/d/11v0X6VNltAN7mASLn2EugM4cBHfz2gNK/view?usp=drive_link

**Reply by Xanashi** [reply]
I've requested permission to download the file

**Comment by Xanashi** [comment]
Thank you! I got the file. 
I'll create an issue for this particular feature and will hopefully be able to implement it soon.

**Reply by Xanashi** [reply]
Here is the new ticket ( #73)

**Comment by Xanashi** [comment]
Should be fixed in #82
